delta/rdf/src/main/scala/io/circe/jakartajson/package.scala

```diff
@@ -0,0 +1,95 @@
+package io.circe
+
+import io.circe.syntax.EncoderOps
+import jakarta.json.JsonValue.{ValueType => JakartaValueType}
+import jakarta.json.{Json => JakartaJson, JsonNumber => JakartaJsonNumber, JsonObject => JakartaJsonObject, JsonString, JsonValue => JakartaJsonValue}
+
+import scala.jdk.CollectionConverters._
+import java.math.{BigDecimal => JBigDecimal}
+
+/**
+  * Support for converting between Jakarta Json and Circe Based on
+  * https://github.com/circe/circe-jackson/blob/master/shared/src/main/scala/io/circe/jackson/package.scala
+  */
+package object jakartajson {
```

Comment:
Nexus and Titanium use different json libraries so this is needed to do a conversion between those.
Before that Nexus was just getting back to a raw string and parse things again with the target library. This had a performance overhead which is now eliminated.